### PR TITLE
Update ubiquity swoole & workerman

### PR DESCRIPTION
**Pull request by jcheron**
#### Changed
- Swoole worker_num (x1->x2)
- Workerman worker_num (x3->x4)

### CODE REVIEW

File: frameworks/PHP/ubiquity/.ubiquity/_swoole.php [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-$swooleServer=new \Ubiquity\servers\swoole\SwooleServer();
+$swooleServer = new \Ubiquity\servers\swoole\SwooleServer();
 $swooleServer->init($config, __DIR__);
-require ROOT.'config/swooleServices.php';
-$swooleServer->run($sConfig['host'],$sConfig['port']);
\ No newline at end of file
+require ROOT . 'config/swooleServices.php';
+$swooleServer->run($sConfig['host'], $sConfig['port'], [
+	'worker_num' => \swoole_cpu_num() * 2,
+	'reactor_num' => \swoole_cpu_num() * 2
```
Comment (vasily-kartashov) [inline frameworks/PHP/ubiquity/.ubiquity/_swoole.php]: @jcheron, do you see any performance improvements with this or is it speculation?

Comment (jcheron) [reply frameworks/PHP/ubiquity/.ubiquity/_swoole.php]: > @jcheron, do you see any performance improvements with this or is it speculation?

I think it's better, to be confirmed...

Comment (joanhey) [reply frameworks/PHP/ubiquity/.ubiquity/_swoole.php]: If it is faster, then it is possible to have better coroutine implementation.
And theoretically will be slower in the plaintext and json tests.

**Comment by joanhey** [comment]
I want to test workerman x4 too, and aslo x5.
But I don't want it mixed with other changes in the same run.
Later we will try it for the platform.

**Comment by vasily-kartashov** [comment]
@jcheron quite impressive results on update test.
https://www.techempower.com/benchmarks/#section=test&runid=e0e5d81b-7177-43a3-8e71-72680544dd50&hw=ph&test=update&l=zik073-v
Do you mind sharing the secret of what you're doing differently from the rest?

**Comment by jcheron** [comment]
Hi @vasily-kartashov 
It's no secret, open source helps us to improve our productions.
https://github.com/TechEmpower/FrameworkBenchmarks/blob/227880491e47718f4d6a0ba380948ee2b0df5b7c/frameworks/PHP/ubiquity/app/controllers/SwooleDb.php#L33-L44

**Ubiquity** makes a bulk-update for each http request.
Only one db query is required to update, which is different from several queries in a transaction.
Note that at this high concurrency level, it only works with pgsql, and not at all with Mysql.

**Pull request merged**